### PR TITLE
Fügt neue Farb-Funktionen hinzu

### PR DESCRIPTION
**Pull request by D3nnisH**
- rgba(Color, alpha)
- alpha(Color) / opacity(Color)
- opacify(Color, amount) / fade_in(Color, amount)
- transparentize(Color, amount) / fade_out(Color, amount)
- mix(Color, Color, weight?)

### CODE REVIEW

File: src/main/java/org/serversass/Functions.java [hunk 1]
```diff
@@ -86,6 +96,30 @@ public static Expression lighten(Generator generator, FunctionCall input) {
         return changeLighteness(color, increase);
     }
 
+    /**
+     * Returns the alpha value of the gioven color
```
Comment (sabieber) [inline src/main/java/org/serversass/Functions.java]: gioven -> given

File: src/main/java/org/serversass/Functions.java [hunk 2]
```diff
@@ -124,4 +158,69 @@ public static Expression desaturate(Generator generator, FunctionCall input) {
...
+    /**
+     * Increases the opacity of the given color by the given amount.
+     *
+     * @param generator the surrounding generator
+     * @param input     the function call to evaluate
+     * @return the result of the evaluation
+     */
+    public static Expression opacify(Generator generator, FunctionCall input) {
+        Color color = input.getExpectedColorParam(0);
+        float decrease = input.getExpectedFloatParam(1);
```
Comment (sabieber) [inline src/main/java/org/serversass/Functions.java]: decrease -> increase ?

File: src/main/java/org/serversass/Functions.java [hunk 3]
```diff
@@ -124,4 +158,69 @@ public static Expression desaturate(Generator generator, FunctionCall input) {
...
+     * @param generator the surrounding generator
+     * @param input     the function call to evaluate
+     * @return the result of the evaluation
+     */
+    public static Expression fade_out(Generator generator, FunctionCall input) {
+        return opacify(generator, input);
+    }
+
+    /**
+     * Decreases the opacity of the given color by the given amount.
```
Comment (sabieber) [inline src/main/java/org/serversass/Functions.java]: wrong javadoc?

Comment (D3nnisH) [reply src/main/java/org/serversass/Functions.java]: ne basst

File: src/main/java/org/serversass/ast/Color.java [hunk 4]
```diff
@@ -24,10 +26,15 @@
      */
     public static final double EPSILON = 0.001;
 
+    public static final NumberFormat ALPHA_NUMBERFORMAT = NumberFormat.getNumberInstance(Locale.ROOT);
```
Comment (andyHa) [inline src/main/java/org/serversass/ast/Color.java]: nicht en?

Comment (andyHa) [reply src/main/java/org/serversass/ast/Color.java]: is number format threadsafe / reusable?

**Comment by meg1502** [comment]
cool stuff 👍

**Pull request merged**